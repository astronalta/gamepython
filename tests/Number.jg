# jgtest exec on

main() Int {
    Io::println(0x00010.str)
    Io::println(0x09.str)
    Io::println(0xDEADBEEF.str) 
    ret 0
}
