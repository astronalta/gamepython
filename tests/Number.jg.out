16
9
3735928559
